interface/src/Application.cpp

```diff
@@ -1139,7 +1139,7 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-SemiBold.ttf");
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-Regular.ttf");
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-Medium.ttf");
-    _window->setWindowTitle("Project Athena");
+    _window->setWindowTitle("Vircadia Interface");
```

Comment:
"Vircadia" or "Vircadia Interface"?
Below, also.

Reply:
Just Vircadia for now.